specification/cognitiveservices/data-plane/Language/preview/2022-04-01-preview/examples/SuccessfulAnalyzeConversationTextJobsTaskSubmitRequest.json

```diff
@@ -0,0 +1,58 @@
+{
+  "parameters": {
+    "Ocp-Apim-Subscription-Key": "{API key}",
+    "api-version": "2022-04-01-preview",
+    "Endpoint": "{Endpoint}",
+    "jobId": "{Job ID}",
+    "body": {
+      "displayName": "Analyze conversations from xxx",
+      "analysisInput": {
+        "conversations": [
+          {
+            "id": "20220101meeting",
+            "language": "en",
+            "modality": "text",
+            "domain": "neutral",
```

Comment:
I think this was `"general"` somewhere else.

Reply:
generic. updated